spec/services/person_constructor_spec.rb

```diff
@@ -70,14 +70,13 @@
       expect(person.location.zip_code).to eq '12345'
     end
 
-    it "rejects bad zip codes" do
-      params = { zip_code: '1231' }
-      expected_params = { zip_code: nil }
-      stub_person_finder(expected_params)
+    it "passes bad zips through to the model" do
+      params = { zip_code: '123456' }
+      stub_person_finder(params, found: false)
 
-      person = PersonConstructor.build(params)
+      PersonConstructor.build(params)
 
-      expect(person.location.zip_code).to be nil
+      expect(PersonFinder).to have_received(:new).with(params)
```

Comment:
this was the quick way to test that the params were working correctly. It's too closely tied to the implementation of PersonConstructor, for my taste, though. Any objection to creating a `PersonParamNormalizer` class? It will leave `PersonConstructor` pretty sparse, but I don't think that's a problem.